client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -141,7 +141,7 @@ const CellText = React.memo(
         label: (
           <>
             {labelFromString(added)}
-            <NonBreakingSpace />— {labelFromString(removed)}
+            <NonBreakingSpace />– {labelFromString(removed)}
```

Comment:
While in the area I wonder if I can sell a few more changes? It seems to me that using zero width spaces around the dash or possibly one of the narrow spaces mentioned in https://en.wikipedia.org/wiki/Whitespace_character#Hair_spaces_around_dashes would be better than the full space we now have.

If I could suggest anything, it would be `{labelFromString(added)}&hairsp;&ndash;&hairsp;{labelFromString(removed)}`